tests/jenkins/install.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/bash
+
```

Comment:
Add `set -e` to break on error

Reply:
I didn't know about this. Thank you for the addition!